src/python/pants/backend/python/goals/publish_test.py

```diff
@@ -166,5 +172,40 @@ def test_skip_twine(rule_runner, packages) -> None:
 
     # Skip twine globally from config option.
     rule_runner.set_options(["--twine-skip"])
-    result = rule_runner.request(PublishProcesses, [fs._request(packages)])
+    result = request_publish_processes(rule_runner, packages)
     assert len(result) == 0
+
+
+@pytest.mark.parametrize(
+    "options, cert_arg",
+    [
+        pytest.param(
+            [],
+            None,
+            id="No ca cert",
+        ),
+        pytest.param(
+            ["--twine-ca-certs-path={}"],
+            "--cert=ca_certs.pem",
+            id="[twine].ca_certs_path",
+        ),
+        # This test needs a working ca bundle to work. Verified manually for now.
+        # pytest.param(
+        #     ["--ca-certs-path={}"],
+        #     "--cert=ca_certs.pem",
+        #     id="[GLOBAL].ca_certs_path",
+        # ),
```

Comment:
What's up with this?

Reply:
I don't have a ca bundle to provide to the test to use.. as the test downloads files, so the value I provide must work for that... or I've missed something. But I tried with an empty file, but that blew up ;)

Reply:
Given that you're not actually running the `Process`, just setting it up, can you create a fake file? I'm not following why this is any different than `--twine-ca-certs-path

Reply:
It fails on this one:
```
E           pants.engine.process.ProcessExecutionFailure: Process 'Building twine.pex with 1 requirement: twine==3.4.2' failed with exit code 1.
```

Reply:
```
WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLError(136, '[X509] no certificate or crl found (_ssl.c:4140)'))': /simple/twine/
```

Reply:
Ah I found this:

https://github.com/pantsbuild/pants/blob/458e754a27c16fa0da82d98e4a14c9472297907c/src/python/pants/engine/fs_test.py#L897-L928

Reply:
Cool. But I don't think that helps here. I need to provide a cert bundle that works against public pypi. So, most likely would be to fish the file out from the system we're running on and copy that to a custom location, or something like that. (Or fish the bundle out of the `certifi` library..)